test/integration/crcsuite/util.go

```diff
@@ -22,14 +23,22 @@ func DeleteCRC() error {
 // Remove CRC home folder ~/.crc
 func RemoveCRCHome() error {
 
-	err := os.RemoveAll(CRCHome)
+	keepFile := filepath.Join(CRCHome, ".keep")
 
-	if err != nil {
-		fmt.Printf("Problem deleting CRC home folder %s.\n", CRCHome)
-		return err
-	}
+	_, err := os.Stat(keepFile)
+	if err != nil { // cannot get keepFile's status
+		err = os.RemoveAll(CRCHome)
 
-	fmt.Printf("Deleted CRC home folder %s.\n", CRCHome)
-	return nil
+		if err != nil {
+			fmt.Printf("Problem deleting CRC home folder %s.\n", CRCHome)
+			return err
+		}
 
+		fmt.Printf("Deleted CRC home folder %s.\n", CRCHome)
+		return nil
+
+	} else { // keepFile exists
+		fmt.Printf("Folder %s not removed as per request: %s present.\n", CRCHome, keepFile)
```

Comment:
I think I would error out and tell the user to delete/move away ~/.crc themselves as I expect this is going in the way of the tests?

Reply:
It shouldn't break tests. Just the test result will mean something different (as the clean install won't be tested). 

But I think I now understand what you wanted -- essentially just be given a chance to back up your folder before running the tests. I changed that part accordingly. I didn't provide a message for the user, as if they have a `.keep` file in the folder, they know why they put it there. WDYT?